src/store/menu/menuSlice.ts

```diff
@@ -0,0 +1,32 @@
+import { createSlice } from '@reduxjs/toolkit';
+import type { RootState } from '../store';
+
+const initialState = {
+  isProfileMenuOpen: false,
+  isMobileMenuOpen: false,
+};
+
+export const menuSlice = createSlice({
+  name: 'menu',
+  initialState,
+  reducers: {
+    handleMobileMenuChange: (state) => {
+      const stateValue = { ...state };
+      stateValue.isMobileMenuOpen = !state.isMobileMenuOpen;
+
+      return stateValue;
+    },
+    handleProfileMenuChange: (state) => {
```

Comment:
I want to know from the function's name what it does.

handle -> toggle

Reply:
add correction